src/js/utils/dom.js

```diff
@@ -1,15 +1,59 @@
 import { trim } from 'utils/strings';
 import { isString, contains, difference, isBoolean } from 'utils/underscore';
 
+const forEach = Array.prototype.forEach;
+let parser;
+
 export function hasClass(element, searchClass) {
     return element.classList.contains(searchClass);
 }
 
-// Given a string, convert to element and return
 export function createElement(html) {
-    const newElement = document.createElement('div');
-    newElement.innerHTML = html;
-    return newElement.firstChild;
+    return htmlToParentElement(html).firstChild;
+}
+
+export function replaceHtml(element, html) {
```

Comment:
Which is better?
- [ ] Rename to `replaceInnerHtml`
- [ ] Leave it as `replaceHtml`